docs/Configuration/fleet-server-configuration.md

```diff
@@ -752,7 +752,11 @@ The bcrypt cost to use when hashing user passwords.
 
 ##### auth_salt_key_size
 
-The key size of the salt which is generated when hashing user passwords.
+The key size of the salt which is generated when hashing user passwords. Note: Fleet uses the
+`bcrypt` hashing algorithm for hashing passwords, which has a [72 character input limit](https://en.wikipedia.org/wiki/Bcrypt#Maximum_password_length). This means
+that the plaintext password (i.e. the password input by the user) length + the value of
+`auth_salt_key_size` cannot exceed 72. In the default case, the max length of a plaintext password
+is 48 (72 - 24).
```

Comment:
Better separate to a doc PR.

Reply:
Ah gotcha, I can pull that out into a separate PR.

Reply:
@rachaelshaw actually wound up reviewing here, so keeping it in this PR